src/image.py

```diff
@@ -1,4 +1,6 @@
 from pathlib import Path
+from scipy.signal import find_peaks
+import numpy
```

Comment:
Why to change the standard convention to use numpy under the `np` alias?

```suggestion
import numpy as np
```

Reply:
didn't know that is a standard :p i'll change it